### PR TITLE
chore: New Crowdin updates

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

**Comment by lipis** [comment]
@ad1992 can you ignore the changelog for this branch?

File: .github/workflows/changelog-check.yml [hunk 1]
```diff
@@ -2,9 +2,10 @@ name: Changelog in sync for packages
 
 on:
   push:
-    branches:
-      - master
   pull_request:
+    branches-ignore:
+      - l10n_master
+      - dependabot/*
```
Comment (lipis) [inline .github/workflows/changelog-check.yml:8]: how do we make it to ignore this branch (and the others of course)

File: .github/workflows/changelog-check.yml [hunk 2]
```diff
@@ -2,8 +2,9 @@ name: Changelog in sync for packages
 
 on:
   push:
-    branches:
-      - master
+    branches-ignore:
+      - "l10n_master"
+      - "dependabot/*"
```
Comment (lipis) [inline .github/workflows/changelog-check.yml:7]:
```suggestion
      - "dependabot/**"
```

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

File: .github/workflows/changelog-check.yml [hunk 3]
```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
```
Comment (lipis) [inline .github/workflows/changelog-check.yml:6]: I think once it will merged to paster.. it will fail the changelog there if we leave it as is.

Comment (ad1992) [reply .github/workflows/changelog-check.yml:6]: that shouldn't happen since we want it to run in master branch though...

Comment (lipis) [reply .github/workflows/changelog-check.yml:6]: If the PR is green, master should be green.. it will be too late if the error is on master :)

**Pull request merged**